controllers/validationController.js

```diff
@@ -38,17 +38,31 @@ exports.validateNameAndEmail = [
 exports.validateAdminPassword = (req, res, next) => {
 
     const adminPassword = config.get('settings.adminAccounts');
-    // console.log("adminPassword", adminPassword);
     const password = req.query.password
-    // console.log("password", password);
-    if (password === adminPassword) {
-        // then good to go to players  
+    
+    // get headers authorization
+    const authheader = req.headers.authorization;
+    console.log('req.headers.authorization', authheader);
+    // and distract only token password
+    if (typeof authheader !== 'undefined') {
+        const tokenPass = authheader.split(' ')[1]
+        // console.log("tokenPass", tokenPass);
+        // get token password from frontend
+        if (password === adminPassword || tokenPass === adminPassword) {
```

Comment:
`password === adminPassword ||` should be gone from here because it will be handled in the code below.